client/src/main/java/io/spine/client/VersionColumn.java

```diff
@@ -24,37 +24,52 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.spine.server.entity.storage;
+package io.spine.client;
 
 import io.spine.core.Version;
-import io.spine.query.ColumnName;
-import io.spine.query.CustomColumn;
-import io.spine.server.entity.Entity;
 
 /**
- * A column of an entity to store the {@linkplain Entity#version() entity version}.
+ * A column of an entity holding the entity version.
+ *
+ * <p>Extends the client-side query language to provide an ability to query entities
+ * by their versions.
+ *
+ * <pre>
+ *  Query query =
+ *        ReportView.query()
+ *                    // ...
+ *                  .where(Version.is(), previousVersion)
+ *                  .build(...);
+ *  ... = execute(query);
+ * </pre>
+ *
+ * <p>This type is a singleton.
  */
-final class VersionColumn extends CustomColumn<Entity<?, ?>, Version> {
+public class VersionColumn extends EntityLifecycleColumn<Version> {
```

Comment:
Let's add the `nameAsString` static method for this column as we have in `Archived` and `Deleted`.

Reply:
We should also make it `final`.